kuksa-client/kuksa_client/__main__.py

```diff
@@ -43,10 +43,16 @@
 from kuksa_client import KuksaClientThread
 from kuksa_client import _metadata
 
-DEFAULT_SERVER_ADDR = "grpc://127.0.0.1:55555"
-
 scriptDir = os.path.dirname(os.path.realpath(__file__))
 
+DEFAULT_SERVER_ADDR = os.environ.get("SERVER_ADDR", "grpc://127.0.0.1:55555")
+DEFAULT_LOGGING_CONFIG = os.environ.get("LOGGING_CONFIG", os.path.join(scriptDir, 'logging.ini'))
+DEFAULT_TOKEN_OR_TOKENFILE = os.environ.get("TOKEN_OR_TOKENFILE", None)
+DEFAULT_CERTIFICATE = os.environ.get("CERTIFICATE", None)
```

Comment:
What is our view on mutual authentication? AFAIK we do not currently support it, as broker does not check/request client cert. I would prefer if we either spend some effort to support it, or conclude that it likely never will be supported and remove all client certs

Reply:
I would vote for removing all that. I don't think they even enable additional security goals over authorisation tokens, which also need to be signed by a trusted authority.
The only advantage is, that TLS cleint auth kicks in "earlier" in the stack on transport level, but even though: We did not have it implemented and it is not on the roadmap, so let's rather clean up here. IMO not much we loose if at a later point we DO decide to implement this.